templates/base.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+
+<html class="no-js{% block extra_html_class %}{% endblock %}" lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>{% block title %}{{ title }}{% endblock %} | Kubeflow News</title>
+
+    <script>
+      var html = document.documentElement
+      html.classList.replace("no-js", "js");
```

Comment:
why is that?

Reply:
I carried it over as boilerplate from another project - I thought we'd need it eventually, but I guess we can introduce it later if we do need it. I'll remove.